### PR TITLE
missing #include <algorithm> for std::find_if_not

### PR DESCRIPTION
**Pull request by kroening**

### CODE REVIEW

File: src/java_bytecode/jar_file.cpp [hunk 1]
```diff
@@ -7,10 +7,13 @@ Author: Diffblue Ltd
 \*******************************************************************/
 
 #include "jar_file.h"
+#include "java_class_loader_limit.h"
```
Comment (tautschnig) [inline src/java_bytecode/jar_file.cpp]: Most other files seem to do:
1) The translation-unit specific header file.
2) STL headers
3) Includes from util/
4) Includes from other directories
5) Includes from the current directory.

Are we changing this practice?

File: src/java_bytecode/jar_file.cpp [hunk 2]
```diff
@@ -7,10 +7,13 @@ Author: Diffblue Ltd
...
+
 #include <cctype>
-#include <util/suffix.h>
+#include <algorithm>
```
Comment (tautschnig) [inline src/java_bytecode/jar_file.cpp]: On which platform has this come up?

File: src/java_bytecode/jar_file.cpp (hunk 1 above)
Comment (kroening) [reply src/java_bytecode/jar_file.cpp]: ok, will re-sort

File: src/java_bytecode/jar_file.cpp (hunk 2 above)
Comment (kroening) [reply src/java_bytecode/jar_file.cpp]: Debian 9, gcc 6

**Review by tautschnig — APPROVED** [review]

File: src/java_bytecode/jar_file.cpp [hunk 3]
```diff
@@ -7,9 +7,13 @@ Author: Diffblue Ltd
 \*******************************************************************/
 
 #include "jar_file.h"
+
 #include <cctype>
-#include <util/suffix.h>
+#include <algorithm>
```
Comment (tautschnig) [inline src/java_bytecode/jar_file.cpp]: Nit pick: swap the above two to maintain lexicographic ordering.

Comment (kroening) [reply src/java_bytecode/jar_file.cpp]: done

**Review by thk123 — APPROVED** [review]

**Pull request merged**